seed.d/rules.json

```diff
@@ -1,37 +1,37 @@
 [
     {
-        "rule_id": "cloud_instance_ros_evaluation|CONSUMPTION_MODEL",
+        "rule_id": "ros_instance_evaluation|INSTANCE_IDLE",
         "description":"This instance is idling so consider switching to a consumption model.",
-        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.cloud_region} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.",
-        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.\nYou may also want to consider changing the instance type from {system.instance_type} to one of the following instance types, which fit the load better: {system.candidate_instance_types}",
+        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.",
+        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.{newline}You may also want to consider changing the instance type from {system.instance_type} ({instance_price} USD/hour) to one of the following instance types, which fit the load better: {candidate_string}",
```

Comment:
```suggestion
        "reason": "Based on the host metrics set, we have detected that this {system.instance_type} instance on {system.cloud_provider.upper()} region {system.region} is displaying a higher amount of idling cycles within a 24h time frame. It can be potentially explained by the non-business hours. This idling state may be caused by picking an inappropriate consumption model.",
        "resolution": "Please verify that this instance and its workload is having an appropriate consumption model for this specific workload.{newline}You may also want to consider changing the instance type from {system.instance_type} ({instance_price} USD/hour) to one of the following instance types, which fit the load better: {candidate_string}.",
```